modular_skyrat/modules/customization/game/objects/items/plushes.dm

```diff
@@ -536,3 +536,14 @@
 	description = "<span class='nicegreen'>Warm cozy bee hugs are the best!</span>\n"
 	mood_change = 0
 	timeout = 2 MINUTES
+
+/obj/item/toy/plush/roselia
+	name = "obscene sergal plushie"
+	desc = "A plush recreation of a pink sergal. The chest is extremely padded and the small plush clothes are barely holding themselves together."
+	icon = 'modular_skyrat/master_files/icons/obj/plushes.dmi'
+	icon_state = "plushie_roselia"
+	attack_verb_continuous = list("hugs")
+	attack_verb_simple = list("hug")
+	squeak_override = list('modular_skyrat/modules/emotes/sound/voice/merp.ogg' = 1)
+	young = 1 //No.
```

Comment:
use true/false instead of 1/0

Reply:
oh. all the other plushies in the file use 1. or the ones at the top of the file at least